modules/metastorage/src/main/java/org/apache/ignite/internal/metastorage/impl/raft/MetaStorageSnapshotStorageFactory.java

```diff
@@ -68,6 +68,8 @@ public MetaStorageSnapshotStorageFactory(KeyValueStorage storage) {
         assert configuration != null;
 
         return new RaftMessagesFactory().snapshotMeta()
+                .cfgIndex(configuration.index())
```

Comment:
For Metastorage, these fields are not needed, so now it looks like we are just conforming to a ceremony. It would be nice if we could avoid using these new 2 fields here.

Reply:
discussed in private